storage/aptosdb/src/pruner/state_store/test.rs

```diff
@@ -98,8 +89,19 @@ fn test_state_store_pruner() {
         ));
     }
 
-    // Prune till version=0. This should basically be a no-op
+    // Prune till version=0. This should basically be a no-op. Create a new pruner everytime to
+    // test the min_readable_version initialization logic.
     {
+        let pruner = StatePrunerManager::new(
+            Arc::clone(&aptos_db.state_merkle_db),
+            Arc::clone(&aptos_db.ledger_db),
+            StateMerklePrunerConfig {
+                enable: true,
+                prune_window: 0,
+                batch_size: prune_batch_size,
+                user_pruning_window_offset: 0,
+            },
+        );
```

Comment:
let's make a helper function to make this one line.

Reply:
Done.